### PR TITLE
Update frontend to dotnet7 part2

### PR DESCRIPTION
**Pull request by paule96**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -31,18 +31,20 @@ jobs:
         uses: actions/setup-dotnet@v1.9.0
         with:
           # Optional SDK version(s) to use. If not provided, will install global.json version when available. Examples: 2.2.104, 3.1, 3.1.x
-          dotnet-version: 6.0.x
+          dotnet-version: |
+            7.0.x
+            6.0.x
```
Comment (kirkone) [inline .github/workflows/main.yml:36]: Why both versions?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -25,6 +25,10 @@ stages:
       vmImage: $(vmImageName)
 
     steps:
+    - task: UseDotNet@2
+      inputs:
+        packageType: 'sdk'
+        version: '7.x'
```
Comment (kirkone) [inline azure-pipelines.yml:31]: Same here, why we need 2 versions here?

File: src/CZ.Azure.FileExchange/CZ.Azure.FileExchange.csproj [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="DartSassBuilder" Version="0.2.1-beta" />
+    <PackageReference Include="DartSassBuilder" Version="0.3.0-beta" />
```
Comment (kirkone) [inline src/CZ.Azure.FileExchange/CZ.Azure.FileExchange.csproj:10]: Should I try to remove this dependency?

Would be more complicated but could fix issues with the builds later on.

File: .github/workflows/main.yml (hunk 1 above)
Comment (paule96) [reply .github/workflows/main.yml:36]: becass dart sass only works with dotnet 6. so our source only need dotnet 7.but our build need dotnet 6.

File: azure-pipelines.yml (hunk 2 above)
Comment (paule96) [reply azure-pipelines.yml:31]: see other comment

File: src/CZ.Azure.FileExchange/CZ.Azure.FileExchange.csproj (hunk 3 above)
Comment (paule96) [reply src/CZ.Azure.FileExchange/CZ.Azure.FileExchange.csproj:10]: yeah but maybe not in this PR. we should avoid making this PR to big.

Comment (kirkone) [reply src/CZ.Azure.FileExchange/CZ.Azure.FileExchange.csproj:10]: ok

**Pull request merged**